app/services/container_dashboard_service.rb

```diff
@@ -160,10 +160,10 @@ def ems_utilization
 
     daily_provider_metrics.each do |metric|
       date = metric.timestamp.strftime("%Y-%m-%d")
-      used_cpu[date] += metric.v_derived_cpu_total_cores_used
-      used_mem[date] += metric.derived_memory_used
-      total_cpu[date] += metric.derived_vm_numvcpus
-      total_mem[date] += metric.derived_memory_available
+      used_cpu[date] += metric.v_derived_cpu_total_cores_used if metric.v_derived_cpu_total_cores_used.present?
```

Comment:
- is metric.v_derived_cpu_total_cores_used a string? if so you probably want to test for ```nil?```
- is it ok to be silent in such cases? is it an error or just viable partial information?